packages/wp-source/src/actions.ts

```diff
@@ -93,11 +90,11 @@ const actions: WpSource["actions"]["source"] = {
           });
         }
         // Or it can be "all".
-      } else if (routerRedirections === "all") {
+      } else if (redirections === "all") {
         redirectionPromise = fetch(state.source.url + link, { method: "HEAD" });
```

Comment:
Uhm... `state.source.url` can contain a trailing slash, so maybe this is a problem in servers that do not handle double slashes:

- `state.source.url` = https://domain.com/
- `link` = /some-post
- The fetch is done to https://domain.com//some-post, which could result in a 404 or a 30X in some server configurations.

In Vercel, it returns a 308:

<img width="455" alt="Screenshot 2020-11-23 at 09 10 17" src="https://user-images.githubusercontent.com/3305402/99940049-b5d02600-2d6b-11eb-9637-64773be7d88a.png">

I guess we better handle that case?

```js
fetch(state.source.url.replace(/\/$/, "") + link, { method: "HEAD" });
```

Maybe we need to improve this in the future with a hook, to make sure people don't need to do this over and over.

_By the way, there are more places where this is used. Maybe we should check them all._